src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -390,6 +444,43 @@ function featuresToThree(features, options) {
 /**
  * @module Feature2Mesh
  */
+/**
```

Comment:
Thanks to add example. But these examples don't appear in the documentation must be moved in the `feature2Mesh.convert` documentation.